website/cue/reference/components/sinks.cue

```diff
@@ -209,9 +247,6 @@ components: sinks: [Name=string]: {
 													json: "JSON encoded event."
 												}
 											}
-											if codec == "ndjson" {
```

Comment:
Are the `if batched`s above still relevant given the introduction of the framing options?